README.md

```diff
@@ -65,6 +67,23 @@ You can find the SysV script at `/etc/init.d/schema-registry` or `service schema
 
 You can find the logs at `/var/log/confluent/schema-registry.log`.
 
+### Kafka Connect
+
+If you include the `recipe[confluent::kafka-connect]` this will install the Confluent package, configure and start the
+Kafka connector. This will listen on port 8083 exposing its rest api to control spinning up new connectors.
+
+You can configure the service using the attribtues `node["confluent"]["kafka-connect"]["worker.properties"][...] = ...`.
+Use Confluent's [Kafka Connect doc](http://docs.confluent.io/2.0.0/connect/userguide.html) to figure out the
+appropriate configuration for yourself.
+
+You can find the SysV script at `/etc/init.d/kafka-connect` or `service kafka-connect [start|stop|restart|status]`.
+
+You can find the logs at `/var/log/confluent/kafka-connect.log`.
+
+### Zookeeper (For development purposes only)
+
+If you include the `recipe[confluent::zookeeper]` this will install the Confluent package and start a zookeeper process. It will listen on port 2181. This is a single zookeeper worker and is not recomended for production use. The primary purpose of this recipe is to get everything running inside vagrant as a self contained system without having to run process on the host machine.
```

Comment:
discussed in person. What i have here works ok, just test kitchen is better. Making a note to come back later and possible remove this in favor of using just kitchen over vagrant. But leaving for now if thats ok with @bbaugher